src/pages/stylesheets_page/reserve.css

```diff
@@ -0,0 +1,97 @@
+.reserve_container {
+  position: relative;
+  width: 100%;
+  height: 100vh; /* Adjust the height as needed */
+  background-image: url('https://www.conradvillas.com/uploads/properties/103/ultra-luxury-bespoke-villas-in-bangkok-46297703.jpeg');
+  background-size: cover;
+}
+
+.overlay-green {
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(100, 138, 31, 0.8);
+}
+
+.reserve_container_in {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100vh;
+  align-items: center;
+  justify-content: center;
+}
+
+.reserve_container_in h3 {
+  font-weight: 700 !important;
+}
+
+.reserve_container_in p {
+  font-size: 0.9rem;
+  width: 60vw;
+}
+
+.line {
+  width: 40vw;
+  border-top: 2px solid #a2c82b;
+  margin: 1rem;
+}
+
+.form_container {
+  display: flex;
+  gap: 2rem;
+}
+
+.date-inputs {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.date-inputs input {
+  background-color: #698d1f;
+  color: white;
+  box-sizing: border-box;
+  padding: 0.5rem;
+  border-radius: 1.5rem;
+  border-color: white;
+  text-align: center;
+}
+
+.select_house {
+  background-color: #698d1f;
+  color: white;
+  box-sizing: border-box;
+  padding: 0.5rem;
+  border-radius: 1.5rem;
+  border-color: white;
+  border-width: 2px;
+  text-align: center;
+}
+
+.form_container button {
```

Comment:
### The form misbehaves if the length of the _House name_ is longer. See below:

| If inserted a longer house name in the dropdown ❌ | The problem in displaying the dropdown ❌ |
| ------------- | ------------- |
| ![image](https://github.com/PrangonGhose/Earthbnb-frontend/assets/64170307/d5cf5222-e602-409c-92c4-2c5316037cde)  | ![image](https://github.com/PrangonGhose/Earthbnb-frontend/assets/64170307/330a0a85-b944-4d4e-a361-bddb10bd37cf)  |

### Suggestions:
- The problem is each house name is taking one line only. You can optimize the dropdown so that the options will be shown in a fixed width but in multiple lines if the house name is larger.
- Also after selecting the house name, it is not necessary to change the width of the input box. You can either hide the rest of the house name or display in multiple lines too.